src/dom/index.js

```diff
@@ -35,8 +35,8 @@ export function setAccessor(node, name, old, value, isSvg) {
 		node.className = value || '';
 	}
 	else if (name==='style') {
-		if (!value || isString(value) || isString(old)) {
-			node.style.cssText = value || '';
+		if (!value || typeof value==='string' || typeof old==='string') {
```

Comment:
Why replacing `isString` with `typeof ...` ?

Reply:
`isString` does exactly the same, and nicer to read, in my opinion 😄 .

Reply:
accidentally changed it when i forked from `master`, sorry! :D